### PR TITLE
Create default styles

### PR DESCRIPTION
**Pull request by githubteacher**
Establish just the basic CSS files and defaults.

### CODE REVIEW

**Comment by githubteacher** [comment]
@githubstudent, Any chance you could review this for me and merge in?

**Comment by githubteacher** [comment]
When this ships, it closes out the discussion and objectives listed in #1.

File: styles/defaults.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+body{
+  font-size: 14px;
```
Comment (githubstudent) [inline styles/defaults.css]: Probably could go with a slightly larger font size default like `16px` but this is good for now.

**Comment by githubstudent** [comment]
:+1: :ship: :sparkles:

**Pull request merged**